dashboard/app/views/pd/application/teacher_application/submitted.html.haml

```diff
@@ -6,7 +6,7 @@
 
   %p
     In most regions, applications are accepted on a rolling basis. And in most cases, our local partner will get back
-    to you within two weeks to let you know your application status. If you have not heard back within two weeks,
+    to you within two-four weeks to let you know your application status. If you have not heard back within two weeks,
```

Comment:
I'm *really* trying not to copy edit, but this is hard to read on the site. Could you type out `two to four` rather than using the dash? (I know that's diff from what the spec says)

Reply:
Will do! Good call